contracts/token/ERC20/extensions/ERC20Votes.sol

```diff
@@ -25,7 +26,7 @@ import "../../../utils/cryptography/ECDSA.sol";
  *
  * _Available since v4.2._
  */
-abstract contract ERC20Votes is ERC20Permit {
+abstract contract ERC20Votes is ERC20Permit, IVotes {
```

Comment:
As a general guideline, we tend to put interfaces before contracts in the inheritance list:

```suggestion
abstract contract ERC20Votes is IVotes, ERC20Permit {
```